docs/DeveloperGuide.adoc

```diff
@@ -2048,6 +2048,40 @@ categories and costs.
 .. Test case: `setCategoryBudget c/Fees b/10.00` + `setCategoryBudget c/Bus b/10.00` + `setCategoryBudget c/Bills b/10.00`
 .. Expected: Four of the five added `CategoryIcon` objects should be displayed in `CategoryPanel`.
 
+=== Find expenses
+
+. Find expenses that are in the ExpenseTracker
+
+.. Test case: `find n/lunch` +
```

Comment:
Put a prerequisite that there should be expenses in the list and that the user is logged in